rllib/agents/ars/ars.py

```diff
@@ -84,13 +86,32 @@ def __init__(self,
                  noise,
                  worker_index,
                  min_task_runtime=0.2):
+
+        # Set Python random, numpy, env, and torch/tf seeds.
+        seed = config.get("seed")
+        if seed is not None:
+            # Python random module.
+            random.seed(seed)
+            # Numpy.
+            np.random.seed(seed)
+            # Torch.
+            if config.get("framework") == "torch":
```

Comment:
What about Tensorflow?

Reply:
Is it possible to move it from policy to here?

Reply:
The tensorflow seed has to be set inside the session context, so it cannot be done here, but has to be done further down inside the `[ES|ARS]TFPolicies`.